trnsysGUI/pythonInterface/regimeExporter/exportRegimes.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+import pandas as _pd
+
+
+def exportRegimeTemplate(projectJson, regimeFileName):
+    pumpsAndValvesAndValues = getPumpsAndValvesWithValuesFromJson(projectJson)
+    writeToCsv(pumpsAndValvesAndValues, regimeFileName)
+
+
+def getPumpsAndValvesWithValuesFromJson(projectJson):
+    with open(projectJson, "r", encoding="utf-8") as openFile:
+        jsonValues = json.load(openFile)
+
+    data = {}
+    undesiredBlocks = [".__BlockDct__", "IDs", "Strings"]
+    for block in jsonValues["Blocks"]:
```

Comment:
Get attribute names from serialization model.
Maybe depends on how this evolves in the future.